tuna/driver_bn.py

```diff
@@ -90,6 +90,10 @@ def compute_direction(self):
       raise ValueError("Can't import driver commmand line, \
           one and only one of forw or back must be set")
 
+  def parse_row(self, db_obj):
```

Comment:
Is there a way to create unified function in driver base and perhaps inherit or import to bn and conv? This might trigger a pylint duplicate warning

Reply:
I believe that you need 2+rows to trigger a duplicate code warning, in this case its just 1 row(the function name). The other lines of code in this function are distinct in both child classes, so it does not trigger duplication. The pylint stage in the CI would have let us know by now.